### PR TITLE
Going through SerializableClosure

### PR DESCRIPTION
**Pull request by szepeviktor**
- be explicit
- add spacious spaces
- fix types

### CODE REVIEW

**Comment by szepeviktor** [comment]
See changes without whitespaces: https://github.com/opis/closure/pull/74/files?diff=unified&w=1

File: src/SerializableClosure.php [hunk 1]
```diff
@@ -59,7 +59,9 @@ class SerializableClosure implements Serializable
      */
     protected static $securityProvider;
 
-    /** Array recursive constant*/
+    /**
+     * Array recursive constant
+     */
     const ARRAY_RECURSIVE_KEY = '¯\_(ツ)_/¯';
```
Comment (szepeviktor) [inline src/SerializableClosure.php:65]: `const ARRAY_RECURSIVE_KEY = '🤷';`

:)

**Comment by sorinsarca** [comment]
I'm sorry Viktor, but I cannot merge this PR because it doesn't fix anything. 

- I would like to keep the spacing as is
- I will not use the absolute form for built-in functions
- There are some strange changes like replacing null checks with instanceof checks - I don't want/need them

It is usually good to open an issue before submitting a PR.

**Comment by szepeviktor** [comment]
Okay-okay.

I understand your points.

**Pull request closed without merging**